app/views/idv/inherited_proofing/retrieval.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title t('inherited_proofing.headings.retrieval') %>
+
+<%= render PageHeadingComponent.new.with_content(t('inherited_proofing.headings.retrieval')) %>
+
+<div class="shield-spinner margin-y-4" id='spinner'>
+  <div class='text-center'>
+    <%= image_tag(
+          asset_url('shield-spinner.gif'),
+          alt: t('inherited_proofing.headings.retrieval'),
```

Comment:
Can we add `width` and `height` attributes here to prevent page layout shifts?

Related:

- https://www.aleksandrhovhannisyan.com/blog/setting-width-and-height-on-images/
- https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/

Reply:
I will look into this. I didn't realize putting the sizing on the images was back in style!

Reply:
I believe I've done this the preferred way now?

Reply:
Yep, looks good to me 👍 Thanks